packages/react-router-dom/modules/NavLink.js

```diff
@@ -43,11 +45,13 @@ NavLink.propTypes = {
   className: PropTypes.string,
   activeStyle: PropTypes.object,
   style: PropTypes.object,
-  isActive: PropTypes.func
+  isActive: PropTypes.func,
+  ariaCurrent: PropTypes.oneOf(['page', 'step', 'location', 'true'])
 }
 
 NavLink.defaultProps = {
-  activeClassName: 'active'
+  activeClassName: 'active',
+  ariaCurrent: 'true'
```

Comment:
I suggested `true` because it's the least specific (and therefore the most widely applicable) value, but you could be right. My thought was that in some cases, it's not a `page` that's active like in the case of a nested route (think an onboarding flow that might have multiple steps in one similar view, ergo `step`).

Totally open to some more informed opinions of course, I'm just speculating.